### PR TITLE
Some Gov. Websites Banke

### PR DESCRIPTION
**Pull request by iam-neo**

### CODE REVIEW

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]
The changes are not in alphabetical order. Can you order them in alphabetical order?

**Review by iam-neo — COMMENTED** [review]
Arranged in alphabetical order. 
please check once

File: data/websites.json [hunk 1]
```diff
@@ -440,6 +445,11 @@
       "url": "https://nepaltradeportal.gov.np/",
       "description": "Official Website of Nepal Trade Information Portal"
     },
+    {
+      "name": "Nepalgunj sub metropolitan",
```
Comment (thenishantsapkota) [inline data/websites.json:449]:
```suggestion
      "name": "Nepalgunj Sub Metropolitan"
```

File: data/websites.json [hunk 2]
```diff
@@ -540,5 +550,6 @@
       "url": "https://daorupandehi.moha.gov.np/en",
       "description": "Official Website Of District  Administration Office Of Rupandehi"
     }
+
```
Comment (thenishantsapkota) [inline data/websites.json:553]: Why is there a random space here?

File: data/websites.json [hunk 3]
```diff
@@ -175,6 +175,11 @@
       "url": "https://daobajhang.moha.gov.np/",
       "description": "Office of the District Administration Office of Bajhang"
     },
+    {
+      "name": "District Administration Office of Banke",
+      "url": "https://daobanke.moha.gov.np/en",
```
Comment (thenishantsapkota) [inline data/websites.json:180]: Remove the `/en` at the end of the website here!

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]

**Review by iam-neo — COMMENTED** [review]
Done!

File: data/websites.json [hunk 4]
```diff
@@ -540,5 +550,6 @@
       "url": "https://daorupandehi.moha.gov.np/en",
       "description": "Official Website Of District  Administration Office Of Rupandehi"
     }
+    
   ]
-}
+}
```
Comment (thenishantsapkota) [inline data/websites.json:555]: There's still no new line at the end of the file.
Please format the files using prettier and push the commit again.

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]

**Review by iam-neo — COMMENTED** [review]
File updated, please check at once

**Review by thenishantsapkota — APPROVED** [review]
LGTM!

**Comment by thenishantsapkota** [comment]
Thank you for your contribution @iam-neo!

**Pull request merged**